src/psyclone/transformations.py

```diff
@@ -3269,6 +3246,7 @@ def apply(self, kern, options=None):
             Implicit_Part, Comment
         from fparser.two.utils import walk
         from fparser.common.readfortran import FortranStringReader
+        from psyclone.psyir.nodes import CodeBlock
```

Comment:
Please move this to the top of the file.

Reply:
Done